Refactor modules (#2217)
pyansys/pyaedt: doc/source/Getting_started/Installation.rst

Requirements
~~~~~~~~~~~~
In addition to the runtime dependencies listed in the installation information, PyAEDT

> requires Ansys Electronics Desktop (AEDT) 2021 R2 or later. AEDT Student Version is also supported.

Reviewer: ```suggestion
requires Ansys Electronics Desktop (AEDT) 2021 R2 or later. The AEDT Student Version is also supported.
```
Is there a reason that you think AEDT should be spelled out here? Is this the first occurrence if a reader starts at the beginning of the doc?

Install on CPython from PyPI
~~~~~~~~~~~~~~~~~~~~~~~~~~~~
You can install PyAEDT on CPython 3.7 through 3.10 from PyPI:

.. code:: python

    pip install pyaedt

> PyAEDT is also available on Conda-Forge:

Reviewer: ```suggestion
You can also install PyAEDT from Conda-Forge:
```

.. code:: python

    conda install -c conda-forge pyaedt

Linux support
~~~~~~~~~~~~~

> PyAEDT works with CPython 3.7 through 3.10  in Linux in AEDT 2022 R2 and later.
> However, you must set up the following environment variables:

Reviewer: ```suggestion
PyAEDT works with CPython 3.7 through 3.10 on Linux in AEDT 2022 R2 and later.
However, you must set up the following environment variables:
```

Install from a batch file
~~~~~~~~~~~~~~~~~~~~~~~~~

> AEDT already includes CPython 3.7, which can be used to run PyAEDT,
> then is possible to use it as a virtual environment to run pyaedt.

Reviewer: ```suggestion
AEDT already includes CPython 3.7, which can be used to run PyAEDT.
It is also possible to use CPython 3.7 as a virtual environment to run PyAEDT.
```

Use IronPython in AEDT
~~~~~~~~~~~~~~~~~~~~~~
PyAEDT is designed to work in CPython 3.7+ and supports many advanced processing packages like ``matplotlib``,

> ``numpy`` and ``pyvista``.

Reviewer: ```suggestion
``numpy``, and ``pyvista``.
```

> PyAEDT is designed to work in CPython 3.7+ and supports many advanced processing packages like ``matplotlib``,
> ``numpy`` and ``pyvista``.
> Anyway user can still use PyAEDT in IronPython environment available in AEDT with some limitations.

Reviewer: ```suggestion
PyAEDT is designed to work in CPython 3.7+ and supports many advanced processing packages like ``matplotlib``,
``numpy``, and ``pyvista``. A user can still use PyAEDT in the IronPython environment available in AEDT with some limitations.

```
